docs/deployment/managed-scanning/gitlab.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: gitlab
+append_help_link: true
+title: GitLab
+hide_title: true
+description: Learn how to add a GitLab repository to Semgrep Managed Scans.
+tags:
+  - Beta
+  - Deployment
+  - Semgrep AppSec Platform
+---
+
+import ScanWithSms from "/src/components/procedure/_scan-with-sms.mdx"
+
+# Add a GitLab repository to Semgrep Managed Scans
+
+Add GitLab repositories to your Semgrep organization in bulk without adding or changing your existing CI workflows through **Managed Scans**. 
+
+## Permissions
+
+Before you can add a GitLab repository to Semgrep Managed Scans, you must create a GitLab personal access token (PAT) and provide it to Semgrep. To do so, follow the instructions that apply to you:
+
+- [Connect a source code manager: GitLab Cloud](/deployment/connect-scm#gitlab-cloud)
+- [Connect a source code manager: GitLab self-managed plans](/deployment/connect-scm#gitlab-self-managed-plans)
+
+See [Pre-deployment checklist > Permissions](/deployment/checklist#permissions) for more information about the permissions used by Semgrep.
+
```

Comment:
sorry katie I forgot to mention that user will also need to toggle this switch to setup the gitlab webhooks.

![image](https://github.com/user-attachments/assets/f735c8ae-d3d9-4476-b4a8-951f59207bb4)